sql/expression/function/reverse_repeat_replace.go

```diff
@@ -44,7 +44,9 @@ func (r *Reverse) Eval(
 
 func reverseString(s string) string {
 	r := []rune(s)
-	for i, j := 0, len(r)-1; i < len(r)/2; i, j = i+1, j-1 {
+	l := len(r)
```

Comment:
I guess what @kuba-- commented was about using `i < j` directly as the condition for the `for` loop, you don't need to calculate `l` and `l2`

```go
for i, j := 0, len(r) - 1; i < j; i, j = i+1, j-1 {
        r[i], r[j] = r[j], r[i]
}
```